packages/cli/src/commands/org/resources/push.ts

```diff
@@ -41,6 +41,7 @@ export default class Push extends Command {
   public static flags = {
     ...wait(),
     ...sync(),
+    // TODO: CDX-935 add flag to automatically transfer vault entries when transferable
```

Comment:
`-y` should do it, let's not add a myriad of flags.

Adding a flag, even just a boolean, add quite a lot of  combination of flags, and thus corner case.

We should be mindful of that when adding new flags.